components/charts/StaticChartPreview.tsx

```
@@ -140,6 +140,61 @@ export function StaticChartPreview({
           </div>
         );
 
+      case 'pivot_table':
+        const pivotColors = CHART_TYPE_COLORS.pivot_table || CHART_TYPE_COLORS.table;
```

Biome (2.4.15)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)